Testrunner/run_tests.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+
+class NukeRunner:
+    """Testrunner for nuke.
+
+    This class will handle the passing of arguments to nuke and pytest.
+    """
+
+    TEST_SCRIPT: Path = Path(".")  # TODO decide on a test path
```

Comment:
Yes, this will append pytest to the python path and run the passed test.
That being said... We need to install requirements with the nuke interpreter, else we might get a pytest version which is not compatible with the nuke version.